Gemfile

```diff
@@ -12,6 +12,8 @@ gem 'chartkick'
 gem 'delayed_job'
 gem 'delayed_job_active_record'
 gem 'devise'
+gem 'ffi'
+gem 'ffi-hunspell'
```

Comment:
If you add `ffi-hunspell` then `ffi` will be brought in automatically. Also, `ffi-hunspell` should be in the `test` group, not the main one. This eliminates the need to install it when installing for production (and removes the unnecessary dependency on hunspell in production.).